src/builder.rs

```diff
@@ -861,6 +901,19 @@ builder_constructors! {
     template;
 }
 
+builder_constructors! {
+    // SVG components
+
+    /// Build a
+    /// [`<svg>`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/svg)
+    /// element.
+    svg <> "http://www.w3.org/2000/svg" ;
+    /// Build a
+    /// [`<path>`](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/path)
+    /// element.
+    path <> "http://www.w3.org/2000/svg";
```

Comment:
What about `rect`, `circle`, `ellipse`, etc?

Reply:
Definitely. Wanted to make sure you were happy with the general path before adding more element constructors.